mmv1/third_party/terraform/services/resourcemanager/resource_google_project_service.go.tmpl

```diff
@@ -65,9 +60,11 @@ var renamedServicesByOldAndNewServiceNames = tpgresource.MergeStringMaps(Renamed
 const maxServiceUsageBatchSize = 20
 
 func validateProjectServiceService(val interface{}, key string) (warns []string, errs []error) {
-	bannedServicesFunc := verify.StringNotInSlice(append(ignoredProjectServices, bannedProjectServices...), false)
-	warns, errs = bannedServicesFunc(val, key)
-	if len(errs) > 0 {
+	var errors []error
+	bannedServicesFunc := verify.StringNotInSlice(bannedProjectServices, false)
+	warns, errors = bannedServicesFunc(val, key)
```

Comment:
I think we can continue to just return `errs` directly here? This will change the error message in a way that may be confusing to end users, since we didn't previously express this as "banned services" iirc.